solc/CommandLineInterface.cpp

```diff
@@ -522,6 +522,14 @@ void CommandLineInterface::createFile(string const& _fileName, string const& _da
 	outFile << _data;
 	if (!outFile)
 		BOOST_THROW_EXCEPTION(FileError() << errinfo_comment("Could not write to file: " + pathName));
+	else
+	{
+		if (!isFileCreated)
+		{
+			cout << "Requested files created." << endl;
```

Comment:
If there are multiple files and you get an error on the second one, the messages will be
```
Requested files created.
Could not write to file: ...
```
I think "Requested files created" should be the last message. Perhaps we could also repeat the output directory as part of the message?

Reply:
As per my tests, if write exception comes after first file, it throws you and program exits. So you don't have change to display `Requested files created.` message.

Please check tests details at https://gist.github.com/anurag-git/c1b3d9b92689f0c75093e34f553a926c

Reply:
I am finding it difficult to get a better way for this message. There are so many scenarios to handle such as "overwriting" logic, "Could not write" logic along with normal operations.
In my opinion, by putting this option its creating more confusion atleast to me while testing all the above scenarios.
I am of the view to push the changes without this fix. Please provide your thoughts.

Reply:
ah sorry, you are of course right!

Reply:
What about just moving this message to the end of the main function?